src/components/income/F1099Info.tsx

```diff
@@ -4,25 +4,42 @@ import { Avatar, Box, Button, IconButton, List, ListItem, ListItemAvatar, ListIt
 import { useDispatch, useSelector } from 'react-redux'
 import { Actions, add1099, remove1099 } from '../../redux/actions'
 import { PagedFormProps } from '../pager'
-import { Income1099, TaxesState, Person, PersonRole, Income1099Type, form1099Types } from '../../redux/data'
+import { TaxesState, Person, PersonRole, Supported1099, Income1099Type, form1099Types } from '../../redux/data'
 import DeleteIcon from '@material-ui/icons/Delete'
 import { GenericLabeledDropdown, LabeledInput } from '../input'
 import { Patterns } from '../Patterns'
 
+const showIncome = (a: Supported1099): string => {
+  switch (a.type) {
+    case Income1099Type.INT: {
+      return a.form.income.toString()
+    }
+    case Income1099Type.B: {
+      const ltg = a.form.longTermProceeds - a.form.longTermCostBasis
+      const stg = a.form.shortTermProceeds - a.form.shortTermCostBasis
+      return `L(${ltg}), S(${stg})`
```

Comment:
Can this be changed to `Long term gains: ${ltg}, Short term gains: ${stg}` for clarity?

Reply:
Yep, I need to figure out a better way to format multi line here. Agree it should be clearer.

Reply:
updated this